vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -133,14 +136,24 @@ def parse_endpoint(entity)
       new_result = parse_base_item(entity)
       new_result[:container_groups] = []
 
-      (entity.endpoints || []).each do |endpoint|
-        next unless endpoint.targetRef.try(:kind) == 'Pod'
-        cg = @data_index.fetch_path(
-          :container_groups, :by_namespace_and_name,
-          # namespace is overriden in more_core_extensions and hence needs
-          # a non method access
-          endpoint.targetRef["table"][:namespace], endpoint.targetRef.name)
-        new_result[:container_groups] << cg unless cg.nil?
+      # TODO: remove once kubernetes api v1beta3/v1.0 will be stable
+      if !entity.respond_to?(:subsets) || entity.subsets.nil?
```

Comment:
You can remove the comment and the entire block. We know that we moved to the newer kubernetes.